.circleci/config.yml

```diff
@@ -143,7 +143,8 @@ jobs:
           command: |
              git clone --depth 1 --branch main https://github.com/open-quantum-safe/liboqs.git &&
              export LIBOQS_INSTALLPATH=$(pwd)/.local && cd liboqs && mkdir _build && cd _build &&
-             cmake -GNinja -DCMAKE_INSTALL_PREFIX=$LIBOQS_INSTALLPATH << parameters.CMAKE_ARGS >> .. && ninja install &&
+             export OPENSSL_INSTALL="$(brew --prefix openssl@3.1 || echo "")"
```

Comment:
shouldn't this read "<< parameters.OPENSSL_PREINSTALL >>" instead of the hardcoded "openssl@3.1"?